app/views/reports/regions/_hypertension_patient_coverage.html.erb

```diff
@@ -0,0 +1,85 @@
+<div class="d-flex ai-center jc-between">
+  <p class="mb-8px">
+    Hypertension patient coverage
+    <span class="p-relative t-1px">
+      <i
+        class="fas fa-question-circle fs-14px c-grey-dark"
+        data-toggle="tooltip"
+        data-placement="right"
+        data-trigger="hover click"
+        data-html="true"
+        title="<%= t('hypertension_patient_coverage_copy') %>"
+      >
+      </i>
+    </span>
+  </p>
+  <p class="mb-8px fw-bold">
+    <% if @region.district_region? ||
+      @region.state_region? && @is_population_available_for_all_districts %>
+      <span class="c-green-dark">
+        <%= number_to_percentage(@hypertension_patient_coverage, precision: 0) %>
+      </span>
+    <% else %>
+      ???
```

Comment:
Do we really want question marks here? Something like 'not available' or 'N\A' would be more clear I think.

Reply:
Agreed, good point! Replaced `???` with `N/A`